tests/tests_app/frontend/panel/test_panel_frontend.py

```diff
@@ -0,0 +1,136 @@
+"""The PanelFrontend wraps your Panel code in your LightningFlow."""
```

Comment:
Can you make the test files match the name of the source file

`/a/b/c.py` should be tested in `/a/b/test_c.py`

Reply:
As there are multiple files associated with the Panel tests as opposed to the test of Streamlit I figured it made most sense to collect these in a separate folder.

![image](https://user-images.githubusercontent.com/42288570/184469372-45f5e484-1313-4209-ba9d-d086dc8fda19.png)

Would you reconsider @awaelchli and let me know if you still prefer the change? Thanks

Reply:
Organized it so that panel is mostly self contained. Once the app state watcher apis become more stable and useful for other frontends, we can make them a separate entity :)